policyengine_us/variables/gov/states/la/tax/income/deductions/la_itemized_deductions.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class la_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana itemized deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = [
+        "https://revenue.louisiana.gov/TaxForms/IT540iWEB(2022)D1.pdf",
+        "https://www.legis.la.gov/Legis/Law.aspx?d=101760",
+    ]
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        itemizes = tax_unit("tax_unit_itemizes", period)
+        medical_expenses = tax_unit("medical_expense_deduction", period)
```

Comment:
The law cites [IRC 213(d)](https://www.law.cornell.edu/uscode/text/26/213#d) for the definition of medical _expenses_, not the [medical expense deduction](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/gov/irs/income/taxable_income/deductions/itemizing/medical_expense_deduction.py) which depends on income.
```suggestion
        medical_expenses = add(tax_unit, period, ["medical_expense"])
```